apps/bfd-shared-test-utils/src/main/java/gov/cms/bfd/DatabaseTestApplicationUrl.java

```diff
@@ -0,0 +1,26 @@
+package gov.cms.bfd;
+
+/** DatabaseApplicationUrl sets the application name for a jdbc url. */
+public class DatabaseTestApplicationUrl {
+  /** Database application test name constant value. */
+  public static final String DATABASE_APPLICATION_TEST_NAME = "bfdtestserver";
+  /**
+   * Sets the db url to include the application name if its not a hsqldb.
+   *
+   * @param databaseUrl database url to use
+   * @return formatted string of url with application name
+   */
+  public static String includeApplicationNameInDbUrl(String databaseUrl) {
+    if (databaseUrl.contains("jdbc:postgresql://") && !databaseUrl.contains("ApplicationName=")) {
```

Comment:
Not sure this will work in cases where we might already have a "?" in the URL; don't we really only need to append the ApplicationName to the URL? For example:
`
    if (databaseUrl.contains("jdbc:postgresql://") && !databaseUrl.contains("ApplicationName=")) {
      if (databaseUrl.contains("?")) {
      	return databaseUrl + "&ApplicationName=" + DATABASE_APPLICATION_TEST_NAME;
      } else {
      	return databaseUrl + "?ApplicationName=" + DATABASE_APPLICATION_TEST_NAME;
      }
  }`